src/Plugin/QuickForm/PcscFieldEnrollment.php

```diff
@@ -210,13 +205,13 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $field_values = $values;
     $field_values['type'] = 'pcsc_field';
     $field_values['field'] = $land->id();
-    $field_values['pcsc_start_date'] = strtotime($form_state->getValue('pcsc_start_date'));
+    $field_values['pcsc_start_date'] = $producer->get('pcsc_start_date')->value;
     unset($field_values['practices']);
     $field = PlanRecord::create($field_values);
     $field->save();
 
     // Create a plan record for each practice.
-    foreach ($values['practices'] as $practice_values) {
+    foreach ($values['practices'] ?? [] as $practice_values) {
```

Comment:
Oops, I mean to put this on a separate commit. This just fixes an warning message I got when submitting the field enrollment form without any practices, the array key doesn't exist.